src/Adherent/Certification/CertificationRequestNotifier.php

```diff
@@ -41,7 +41,7 @@ public function sendRefusalMessage(CertificationRequest $certificationRequest):
     {
         $refusalReason = $certificationRequest->isRefusedWithOtherReason()
             ? $certificationRequest->getCustomRefusalReason()
-            : $this->translator->trans($certificationRequest->getRefusalReason())
+            : $this->translator->trans($certificationRequest->getRefusalReasonKey())
```

Comment:
why not just add the key prefix here ? I dont really like to add this into the entity, would be the last file I check to edit it if needed

Reply:
for keeping the translation key in the one place (`getRefusalReasonKey`) and avoiding 3 concatenations with this prefix `certification_request.refusal_reason.`